client/my-sites/people/people-list-item/index.jsx

```diff
@@ -1,36 +1,45 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames';
 import omit from 'lodash/omit';
+import identity from 'lodash/identity';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import CompactCard from 'components/card/compact';
 import PeopleProfile from 'my-sites/people/people-profile';
-import analytics from 'lib/analytics';
 import config from 'config';
+import { recordGoogleEvent } from 'state/analytics/actions';
 
-export default React.createClass( {
+const browserWindow = 'undefined' === typeof window
+    ? window
+    : { scrollTo: identity };
 
-	displayName: 'PeopleListItem',
+export class PeopleListItem extends PureComponent {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
 
-	mixins: [ PureRenderMixin ],
+	static defaultProps = {
+		translate: identity,
+	};
 
-	navigateToUser() {
-		window.scrollTo( 0, 0 );
-		analytics.ga.recordEvent( 'People', 'Clicked User Profile From Team List' );
-	},
+	navigateToUser = () => {
+		browserWindow.scrollTo( 0, 0 );
+		this.props.recordGoogleEvent( 'People', 'Clicked User Profile From Team List' );
```

Comment:
this should work and we can confirm in the Redux DevTools, but it's not entirely semantic to what we are doing. what is the essence of this action? is it "navigating to a user?" If so, let's give it that notion from `connect`. Our components really shouldn't need to think about analytics. They should be free to define and use their own minimal API.

```js
const navigateToUser = dispatch => {
	browserWindow.scrollTo( 0, 0 );
	dispatch( recordGoogleEvent( 'People', 'Clicked User profile from team list' ) );
};

const mapDispatchToProps = ( {
	navigateToUser,
} );
```

we've removed all side-effects from the component and now the only thing it needs to know is, "Hey! I want to navigate to a user," and nothing about the window or about shipping off analytics.

further, I think we can see how with this change and a few small other changes we can turn this component into its own stateless functional component. work for another PR at this point with the diff as big as it is though.